.github/workflows/e2e_tests.yaml

```diff
@@ -114,8 +127,8 @@ jobs:
           scp -r ~/actions-runner/_work/mizar/mizar ciw1:
           scp -r ~/actions-runner/_work/mizar/mizar ciw2:
 
-          ssh ciw1 "cd mizar && make"
-          ssh ciw2 "cd mizar && make"
+          ssh ciw1 "cd mizar && make container"
```

Comment:
CNI will be make in master machine and deploy to worker machines, so fix for CNI will be deployed to worker nodes. To make CNI, it requires to do some environment setup. I think it's reasonable to be done only in master machine. For node machine, maybe keep it as clean as possible then it can catch some real issue if it requires any dependency in worker node but we are not aware of.

Reply:
Since you added make container to make all below, I believe you can just run make instead of `make && make container.`
An improvement suggestion, I think we can just run make once on the master and just copy the mizar folder over to the workers. Right now, it copies over the mizar folder first, then compiles on each machine.

Reply:
It's not "make && make container". It's "cd mizar && make container".
As discussed, we can improve the make process later. I created issue https://github.com/CentaurusInfra/mizar/issues/530 for tracking.

Reply:
Done as requested.